runtime/drivers/repo.go

```diff
@@ -46,3 +47,31 @@ type DirEntry struct {
 	Path  string
 	IsDir bool
 }
+
+// ignoredPaths is a list of paths that are ignored by the parser.
+var ignoredPaths = []string{
+	"/tmp",
+	"/.git",
+	"/node_modules",
+}
```

Comment:
Let's also add `.DS_Store`.

Can you think of others that would be nice?

Reply:
Having editor folders here makes sense to me – don't see any reason people would ever want to edit `.vscode` inside of Rill :)

Reply:
Cool. I will move `.vscode` and `.idea` here from the init.